src/diffusers/utils/testing_utils.py

```diff
@@ -39,6 +39,13 @@
 
 logger = get_logger(__name__)
 
+# Used to test the hub
+USER = "__DUMMY_TRANSFORMERS_USER__"
+ENDPOINT_STAGING = "https://hub-ci.huggingface.co"
+
+# Not critical, only usable on the sandboxed CI instance.
+TOKEN = "hf_94wBhPGp6KrrTH3KDchhKpRxZwd6dmHWLL"
```

Comment:
Those could also be moved to `./tests/others/test_utils.py` as well to have all the "staging logic" at the same place